src/Units/KaijuWarsUnits.java

```diff
@@ -0,0 +1,783 @@
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : extras)
+      kjwModels.unitModels.add(um);
+
+    // Multiplies all movement (and ranges) by N
+    for( UnitModel um : kjwModels.unitModels )
+      um.baseMovePower *= KaijuWarsWeapons.KAIJU_SCALE_FACTOR;
+
+    return kjwModels;
+  }
+
+  public void registerStateTrackers(GameInstance gi)
+  {
+    super.registerStateTrackers(gi);
+
+    StateTracker.instance(gi, UnitTurnPositionTracker.class);
+
+    UnitResurrectionTracker rezzer = StateTracker.instance(gi, UnitResurrectionTracker.class);
+    // Populate resurrection pairs
+    GameReadyModels grms = gi.rules.unitModelScheme.getGameReadyModels();
+    for( UnitModel um : grms.unitModels )
+    {
+      KaijuWarsUnitModel umCast = (KaijuWarsUnitModel) um;
+      if( null == umCast.resurrectsAs )
+        continue;
+
+      // We have a type that resurrects; find the type it rezzes to
+      for( UnitModel rezToModel : grms.unitModels )
+      {
+        if( !rezToModel.name.equals(umCast.resurrectsAs) )
+          continue;
+        rezzer.resurrectionTypeMap.put(um, rezToModel);
+        break;
+      }
+    }
+
+    SuicideAttackTracker kaputnikTracker = StateTracker.instance(gi, SuicideAttackTracker.class);
+    for( UnitModel um : grms.unitModels )
+    {
+      KaijuWarsUnitModel umCast = (KaijuWarsUnitModel) um;
+      if( !umCast.suicideAttack )
+        continue;
+
+      kaputnikTracker.killTypeMap.add(um);
+    }
+  }
+
+  /*
+   * Units I do not plan to implement at this time
+
+   * Freezer (0-counter unit with 1 damage that slows targets by a tile; too complicated and bad)
+   * Food Cart (attracts kaiju within 3 tiles; not implementing kaiju path/targeting mechanics)
+   * Shark Jet (faster Fighter that moves after shooting; messy and either OP or UP)
+   * Cannon of Boom (immobile Rocket equivalent that teleports between labs; ew)
+   */
+
+  /*
+   * Boosts from technologies and tactics; consider making expensive variant units with these boosts
+
+   * tank: +2 counter while on urban
+   * missile: +2 ATK while stationary, 1-2 scoot-n-shoot
+   *  - Giving them the former
+   *  - The latter is hilariously OP, so let's make an "uber missile" that does that?
+   * AA: all attacks get +1 damage for each AA next to target, force air units to land on hit
+   *  - Added the +1; the other is too jank
+   * inf: +2 counter on ROUGH terrain
+   * Radar: +2 science if you end turn within 2 tiles of a Kaiju
+   *  - Should implement as +2 CO energy, but only for opposing Kaiju
+   * bomber: enable hitting air for 1 damage
+   * fighter: +2 counter on empty(plains?)/water
+   *  - Adding
+   * ground: +2 ATK hills, +2 counter on forest/jungle
+   *  - Adding the first
+   * air: +3 move on airport
+   *  - Adding
+
+   * Hover Drives: Gives all ground +1 move and the ability to move on water (Super Z2 doesn't benefit)
+   * Ultimate Autobahn: Gives all ground +2 move if starting on road/bridge
+   */
```

Comment:
These seem like dev notes: are they still relevant, or can we clean this up?

Reply:
They totally are dev notes, but they're relevant since it's the "source" info and might be relevant comparisons if KaijuWars itself ever gets updated and we want to match.